flaskr_tests.py

```diff
@@ -44,18 +44,36 @@ def logout(self):
         """Log the admin user out."""
         return self.app.get('logout', follow_redirects=True)
 
-    def test_login_logout(self):
-        """Login/logout test."""
+
+class FlaskrTestCase(DBTestCase):
+    """Main application test class."""
+
+    def setUp(self):
```

Comment:
If you don't do anything else in the child class, there is no need overriding the method. It will work the way it is defined in the parent class.